docs/io/fits/usage/unfamiliar.rst

```diff
@@ -350,44 +340,52 @@ encapsulate the data into the group data structure, and use :class:`GroupsHDU`
 to create the HDU itself::
 
     >>> # Create the image arrays. The first dimension is the number of groups.
-    >>> imdata = numpy.arange(100.0, shape=(10, 1, 1, 2, 5))
+    >>> imdata = np.arange(150.0).reshape(10, 1, 1, 3, 5)
     >>> # Next, create the group parameter data, we'll have two parameters.
     >>> # Note that the size of each parameter's data is also the number of
     >>> # groups.
     >>> # A parameter's data can also be a numeric constant.
-    >>> pdata1 = numpy.arange(10) + 0.1
+    >>> pdata1 = np.arange(10) + 0.1
     >>> pdata2 = 42
     >>> # Create the group data object, put parameter names and parameter data
     >>> # in lists assigned to their corresponding arguments.
     >>> # If the data type (bitpix) is not specified, the data type of the
     >>> # image will be used.
-    >>> x = fits.GroupData(imdata, parnames=['abc', 'xyz'],
-    ...                    pardata=[pdata1, pdata2], bitpix=-32)
+    >>> x = fits.GroupData(imdata, bitpix=-32,
+    ...                    parnames=['abc', 'xyz', 'xyz'],
+    ...                    pardata=[pdata1, pdata2, pdata2])
     >>> # Now, create the GroupsHDU and write to a FITS file.
     >>> hdu = fits.GroupsHDU(x)
     >>> hdu.writeto('test_group.fits')
     >>> hdu.header
-    SIMPLE =            T / conforms to FITS standard
-    BITPIX =          -32 / array data type
-    NAXIS  =            5 / number of array dimensions
-    NAXIS1 =            0
-    NAXIS2 =            5
-    NAXIS3 =            2
-    NAXIS4 =            1
-    NAXIS5 =            1
-    EXTEND =            T
-    GROUPS =            T / has groups
-    PCOUNT =            2 / number of parameters
-    GCOUNT =           10 / number of groups
-    PTYPE1 = 'abc '
-    PTYPE2 = 'xyz '
-    >>> print(hdu.data[:2])
```

Comment:
That one through a weird TypeError (None is not sliceable), that's why I removed it.